openapi-cli/src/test/resources/ballerina-to-openapi/expected_gen/header_scenario12.yaml

```diff
@@ -0,0 +1,27 @@
+openapi: 3.0.1
+info:
+  title: PayloadV
+  version: 0.0.0
+servers:
+  - url: "{server}:{port}/payloadV"
+    variables:
+      server:
+        default: http://localhost
+      port:
+        default: "9090"
+paths:
+  /ping:
+    get:
+      operationId: operation_get_/ping
+      parameters:
+        - name: headerValue
+          in: header
+          schema:
+            type: integer
+            format: int32
```

Comment:
Are we mapping into `int32` by default? shouldn't we consider other int subtypes?

Reply:
We are not mapping intentionally to subtypes. When we create the new IntegerSchema it will automatically assign int32 as its format within the OAS parser. since ballerina `int` considers as `int64` format I will set the format into int64. :)